app/jobs/wakes/update_facebook_metrics_job.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+class Wakes::UpdateFacebookMetricsJob < ActiveJob::Base
+  queue_as :wakes_metrics
```

Comment:
@benhutton I am wondering if we should create a separate queue here. 

We should probably also move this job to the app. As long as Wakes provides an API to update metrics, it might be fair to consider how an app makes asynchronous updates as it's own concern.